packages/salesforcedx-vscode-soql/src/queryResultsView/media/queryDataViewController.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+(function() {
+  const vscode = acquireVsCodeApi();
```

Comment:
This is the JS file that runs in the webview. Any logic to communicate with the extension or control table behavior happens here.